src/cow-react/modules/advancedOrders/containers/AdvancedOrdersWidget/index.tsx

```diff
@@ -52,6 +54,11 @@ export function AdvancedOrdersWidget() {
     settingsWidget: <div></div>,
     bottomContent: (
       <>
+        <styledEl.Row>
+          <NumberOfParts />
+          <NumberOfParts />
```

Comment:
Is it intentional to add it twice?

Reply:
It's used as a placeholder. In the next PR it is replaced by the slippage component